wpeverest/user-registration
# Fix - Form Builder RTL layout design issue.

### All Submissions:



### Changes proposed in this Pull Request:

There was a design issue in form builder for site with default language set in RTL layout (urdu).

Closes # .

### How to test the changes in this Pull Request:

1. Set site language to the one that supports RTL layout
2. Check whether the design on form builder is fine.
3. Also verify whether this fix has not altered normal design on other languages.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Form Builder RTL layout design issue.
